packages/api/src/graphql/member-plan/member-plan.private-mutation.ts

```diff
@@ -29,3 +29,18 @@ export const createMemberPlan = (
     data: {...input, modifiedAt: new Date()}
   })
 }
+
+export const updateMemberPlan = (
+  id: string,
+  input: Omit<Prisma.MemberPlanUncheckedUpdateInput, 'modifiedAt' | 'createdAt'>,
+  authenticate: Context['authenticate'],
+  memberPlan: PrismaClient['memberPlan']
+) => {
+  const {roles} = authenticate()
+  authorise(CanCreateMemberPlan, roles)
```

Comment:
This should probably be "CanUpdateMemberPlan", if exists.

Reply:
Only peer profile and settings currently have separate permissions for updating.